docs/sources/dashboards/time-range-controls.md

```diff
@@ -105,7 +105,7 @@ For more advanced time settings, click the **Dashboard settings** (gear) icon at
   - **Local browser time -** The time zone configured for the viewing user browser is used. This is usually the same time zone as set on the computer.
   - Standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), including UTC.
 - **Auto-refresh -** Customize the options displayed for relative time and the auto-refresh options. Entries are comma separated and accept any valid time unit.
-- **Now delay** Override the `now` value by entering a time delay. Most commonly, this feature is used to accommodate known delays in data aggregation to avoid null values.
+- **Now delay -** Override the `now` value by entering a time delay. Most commonly, this feature is used to accommodate known delays in data aggregation to avoid null values.
```

Comment:
Where do we define "now"? 

Alternate suggestion: 
Override the specified time by adding a delay. Use this option to accommodate known delays in data aggregation to avoid null values.

Reply:
Also, should it be "Delay now" instead of "Now delay"?

Reply:
👍🏻